Readme.md

```diff
@@ -0,0 +1,494 @@
+
+## Create
+
+With all the heavy lifting done earlier ( 10 lines of code for connecting), adding a new document to the database is really simple and just one
+line of code with `insertOne`. So let's create a new file called `Create.kt`, which will contain all the create operations.
+
+```kotlin
+suspend fun addItem(database: MongoDatabase) {
+
+    val collection = database.getCollection<Restaurant>(collectionName = "restaurants")
+    val item = Restaurant(
+        id = ObjectId(),
+        address = Address(
+            building = "Building", street = "street", zipcode = "zipcode", coord =
+            listOf(Random.nextDouble(), Random.nextDouble())
+        ),
+        borough = "borough",
+        cuisine = "cuisine",
+        grades = listOf(
+            Grade(
+                date = Date(System.currentTimeMillis()),
+                grade = "A",
+                score = Random.nextInt()
+            )
+        ),
+        name = "name",
+        restaurantId = "restaurantId"
+    )
+
+    collection.insertOne(item).also {
+        println("Item added with id - ${it.insertedId}")
+    }
+}
+```
+
+And the output on the console would be
+
+![insert one](https://images.contentstack.io/v3/assets/blt39790b633ee0d5a7/blt1d124cbfb185d7d6/648ae0b2359ef0161360df47/Screenshot_2023-06-15_at_10.49.33.png)
+
+If we want to add many documents to the collection we can use `insertMany`, which is recommended over running `insertOne` in a loop.
+
+```kotlin
+suspend fun addItems(database: MongoDatabase) {
+    val collection = database.getCollection<Restaurant>(collectionName = "restaurants")
+    val newRestaurants = collection.find<Restaurant>().first().run {
+        listOf(
+            this.copy(
+                id = ObjectId(), name = "Insert Many Restaurant first", restaurantId = Random
+                    .nextInt().toString()
+            ),
+            this.copy(
+                id = ObjectId(), name = "Insert Many Restaurant second", restaurantId = Random
+                    .nextInt().toString()
+            )
+        )
+    }
+
+    collection.insertMany(newRestaurants).also {
+        println("Total items added ${it.insertedIds.size}")
+    }
+}
+
+```
+
+![Insert many output](https://images.contentstack.io/v3/assets/blt39790b633ee0d5a7/blt02fc3f33de844c88/648ae1ce2c4f87306c1b12ce/Screenshot_2023-06-15_at_11.02.48.png)
+
+With these outputs on the console, we can say that data has been added successfully. But what if we want to see the object in the database one way
+could be a Read operation, which we would do shortly or use [MongoDB compass](https://www.mongodb.com/products/compass) to view information.
+
+![MongoDB compass](https://images.contentstack.io/v3/assets/blt39790b633ee0d5a7/blt72fd049dd230ea55/648ae40e1fb2d38f0e495940/Screenshot_2023-06-15_at_11.12.23.png)
+
+
+---------------------
+
+## Read
+
+To read the information from the database, we can use the `find` operator and let's begin by reading a random document.
```

Comment:
A `find` operation with a `limit` does not return a random document. It returns the first document in the result set.

Reply:
Yes, I am aware of that. But I am talking from the user's perspective, not from the code. Happy to rephrase it if you feel it's giving the wrong message.

Reply:
rephrased.